src/stapi_fastapi/models/order.py

```diff
@@ -16,7 +16,7 @@ class OrderProperties(BaseModel):
 class Order(Feature[Geometry, OrderProperties]):
     # We need to enforce that orders have an id defined, as that is required to
     # retrieve them via the API
-    id: StrictInt | StrictStr  # type: ignore
```

Comment:
`# type: ignore` here is required to satisfy certain type checkers (pyright) because this change makes `id` incompatible with `Feature` (which has `id: StrictInt | StrictStr | None`) without a default value.